targets/psa_switch/externs/psa_counter.h

```diff
@@ -0,0 +1,69 @@
+/* Copyright 2019-present Derek So
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Derek So (dts76@cornell.edu)
+ *
+ */
+
+
+#ifndef PSA_SWITCH_PSA_COUNTER_H_
+#define PSA_SWITCH_PSA_COUNTER_H_
+
+#define SPEC_ID 0xffffffff
```

Comment:
can we not use a macro for this and use a `static constexpr p4_id_t` variable in the `PSA_Counter` class